inputgraph/inputgraph.go

```diff
@@ -195,7 +196,15 @@ func (l *loader) handleFor(ctx context.Context, forStmt spec.ForStatement) error
 }
 
 func (l *loader) handleWait(ctx context.Context, waitStmt spec.WaitStatement) error {
-	return errors.Wrap(ErrUnableToDetermineHash, "wait not supported")
+	l.hasher.HashString("wait")
+	sort.Strings(waitStmt.Args)
+	l.hasher.HashString(strings.Join(waitStmt.Args, " ")) // TODO: Is this necessary/helpful?
```

Comment:
It might be helpful in case `WAIT` gets args in the future.

Reply:
definitely useful -- I like the sorting of the Args -- that's actually something the existing code doesn't do a great job of.

a command like  `RUN --ssh --no-cache echo this last bits of the args matter`, will simply hash all args in the same order since we use a rather generic approach:

```
func (h *Hasher) HashCommand(cmd spec.Command) {
	dt, err := json.Marshal(cmd)
	if err != nil {
		panic(fmt.Sprintf("failed to hash command: %s", err)) // shouldn't happen
	}
	h.HashBytes(dt)
}
```

later on in the code it makes calls to `parseArgs` -- we could in theory figure out which of the Args represent flags (and can therefore be sorted), and which Args are part of the command can't be changed.

But this is all a rather small improvement to auto-skip in general, and could be pushed down the road.

Reply:
Hmm.. I missed this part. Couldn't sorting lead to strange behavior? `RUN foo bar` being hashed the same as `RUN bar foo`?

Reply:
in the generic sense (like `RUN`, or `COPY`) yes; ~~however it's unlikely that the ordering of the future args (flags) for `WAIT` would matter.~~

actually the order does matter since we could have future flags like `WAIT --key value` rather than `WAIT --key=value`.

best to remove the `sort.Strings(waitStmt.Args)`.

If the user changes any ordering of flags in their Earthfile, I think it's fine for us to report it's a difference -- I don't think it's a use-case which we'll encounter frequently.